Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
+
+open System
+open System.Net.Mail
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.QueriesHandler
+open Book_A_Desk.Domain.Office.Domain
+
+type EmailNotification =
+    {
+        SendEmailNotification: Booking -> unit
+    }
+
+module EmailNotification =
+    let initialize getEmailServiceConfiguration getOffices =         
+        let sendEmailNotification (booking: Booking) =
+            let config = getEmailServiceConfiguration()        
+            let officeName =
```

Comment:
I think this part, where we fetch an office's name, should be done in the Domain, since it's business logic, and tested on it's own in a unit test.

Reply:
To Do: Extract logic to the domain.